web/client/epics/catalog.js

```diff
@@ -72,19 +72,33 @@ import { getResolutions, METERS_PER_UNIT } from "../utils/MapUtils";
 import { describeFeatureType } from '../api/WFS';
 import { extractGeometryType } from '../utils/WFSLayerUtils';
 import { createDefaultStyle } from '../utils/StyleUtils';
+import { removeDuplicateLines } from '../utils/StringUtils';
+
 const onErrorRecordSearch = (isNewService, errObj) => {
+    // Exception text is shown as is while the network errors are shown
+    // with generic error message in the notification
+    let [errorMsg] = castArray(errObj?.error);
+    if (errorMsg) {
+        // Remove any instance of duplicated line string from the exception text
+        errorMsg = removeDuplicateLines(errorMsg);
+    }
+    console.warn("Catalog error", errorMsg);
```

Comment:
please use logError from debug utils instead so this will appear only in debug mode

Reply:
I intended for this error to be shown in the console, so the user can easily identify the cause, if any. Kindly let me know if you still prefer to show it only in debug mode

Reply:
long time ago (when that function have been created), I was asked from @offtherailz to use console log for showing errors only in debug mode. this can be extended to warnings even if in this case is more appropriate using console.error

if we want to change this convention we have to discuss internally within the team. 
so yes, please from now one consider using it

Reply:
I didn't see this PR in detail. 
I can also add some guidelines for general policy, so you can decide what to do:

When an error occours, the application need to handle it, in order to make the user aware of how to proceed, introducing less details as possible, all on UI. The user should never look to the console. 

If the exception is one of the errors that we predicted to happen and maange, this means that it is involved in the normal application workflow, so logging it is  an option only for developers, so in this case a debug-only log makes sense. 

If the error is **not expected** (so it is not one of the errors we predicted to handle), than logging in the console is correct (not masking to the user), so in future we can decide to handle also it. 
The ui in this case should provide a generic error message, with some details if possible.